naloge/2016/dn1/matrix/JureMarkun/slowmatrix.py

```diff
@@ -59,11 +59,11 @@ def multiply(self, left, right):
                     [2, 1, 2, 1, 2, 1, 2],
                     [2, 1, 2, 1, 2, 1, 2],
                     [2, 1, 2, 1, 2, 1, 2]])
```

Comment:
Tole vse zakomentiraj (ali še raje, pobriši in premakni drugam), saj zdaj tvoj program ni sintaktično pravilen.